### PR TITLE
fix: loading until ready

### PR DESCRIPTION
**Pull request by andreiv1992**
This PR...

## Changes

- Added a loading field

## Fixes

- Fix: #1039

## How to test it

- Open monokle with "automatically load last project" on and off

## Screenshots

- 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/PaneManager/PaneManager.tsx [hunk 1]
```diff
@@ -199,62 +202,65 @@ const PaneManager = () => {
...
-                  <HelmPane />
-                </div>
-                <div
-                  style={{
-                    display: leftMenuSelection === 'templates-pane' ? 'inline' : 'none',
-                  }}
-                >
-                  <TemplateManagerPane />
+      {
+        isProjectLoading ? (<Skeleton />) :
```
Comment (erdkse) [inline src/components/organisms/PaneManager/PaneManager.tsx:206]: Too many ternary - I think it makes code reading harder

**Review by erdkse — APPROVED** [review]
LGTM - I think only nested ternary is problematic for readability.

File: src/components/organisms/PaneManager/PaneManager.tsx (hunk 1 above)
Comment (andreiv1992) [reply src/components/organisms/PaneManager/PaneManager.tsx]: true, tried to fix it

**Review by devcatalin — APPROVED** [review]
LGTM

**Pull request merged**